builtin/logical/aws/path_roles.go

```diff
@@ -61,6 +62,11 @@ will be passed in as the Policy parameter to the AssumeRole or
 GetFederationToken API call, acting as a filter on permissions available.`,
 			},
 
+			"default_sts_ttl": &framework.FieldSchema{
```

Comment:
What do you think of naming this just `sts_ttl`? A value on the role will be used unless specified otherwise. The supplied value can be lesser but not greater and this can be documented.

Reply:
Other plugins just call it `ttl`, so `sts_ttl` is a better fit I would say.